### DISCUSSION TITLE
[javascript deep dive] 11장. 원시 값과 객체의 비교

### DISCUSSION THREAD

**Discussion by rladpwl0512** [woowacourse-study/2022-javascript-bank#15] [General]
[javascript deep dive] 11장. 원시 값과 객체의 비교

**Comment by InKyoJeong** [comment]
얕은 복사와 깊은 복사의 차이점은?

**Comment by jin7969** [comment]
객체타입은 왜 변경이 가능할까?

**Comment by jin7969** [comment]
객체는 왜 참조 값을 바라보게 설계했을까?

**Reply by rladpwl0512** [reply]
- 객체타입이 변경가능한 값으로 설계가 된 이유는 무엇일까요? 
- 이런 설계의 단점은 무엇일까요?

**Comment by jin7969** [comment]
153쪽 문제 같이 보기

**Comment by rladpwl0512** [comment]
원시타입의 값과 객체타입의 값을 비교하여 설명해주세요

**Comment by rladpwl0512** [comment]
원시값을 할당한 변수에 새로운 원시값을 재할당하면 생기는 일을 설명해주세요. (Javascript 엔진 내부동작의 관점에서)

**Comment by rladpwl0512** [comment]
값의 불변성이란?

**Comment by rladpwl0512** [comment]
원시값의 가장 큰 특징과, 그로인한 장점은 무엇인가요?

**Comment by rladpwl0512** [comment]
원하는 값에 접근하는 방식에 대해 설명해주세요
- 원시타입
- 객체타입

**Comment by rladpwl0512** [comment]
값을 변경하는 방식에 대해 설명해주세요
- 원시타입
- 객체타입
